README.md

```diff
@@ -48,6 +48,7 @@ If you discover any security related issues, please email <developers@cottacush.
 ## Credits
 
 - Adegoke Obasa goke@cottacush.com
+- Olawale Lawal <wale@cottacush.com>
 - [All Contributors][link-contributors]
```

Comment:
this link does not work, can it be removed.

Reply:
It's just a placeholder for now